src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machinesArray = {new Truck(), new Bulldozer(), new Excavator()};
+        for (Machine m : machinesArray) {
```

Comment:
Let's come up this a better name for the variables)
```suggestion
        for (Machine m : machinesArray) {
```